src/commands/asset-compute/devtool.js

```diff
@@ -87,17 +88,21 @@ class DevToolCommand extends BaseCommand {
                 }
                 return reject(error);
             });
-            server.on('listening', () => onListening(server, randomString));
-            server.on('close', () => {
+            this.server.on('listening', () => onListening(this.server, randomString));
+            this.server.on('close', () => {
                 util.log("Asset Compute Developer Tool Server Stopped");
-                process.exit();
+                // process.exit();
             });
-            process.on('SIGINT', function() {
+            this.onProcessExit(async () => {
```

Comment:
First, I'll define "cleanup" for a server: In this case, it means currently running requests have a chance to finish, aka there is a grace period between reception of a termination signal and the actual termination of the process to do some cleanup.

Do we listen to:
- SIGINT (sent when the user does ctrl+c, cleanup then end process)?
- SIGQUIT (user initiated forced termination, kill immediately)?
- SIGKILL (kill immediately)?
- SIGTERM (give a chance to cleanup, then kill)?

WHich one do we actually handle in the end?

Reply:
`onProcessExit` is here in the BaseCommand and has been tested with the other commands and in `wskdebug` before: https://github.com/adobe/aio-cli-plugin-asset-compute/blob/master/src/base-command.js#L131

This handles SIGINT (ctrl+c by user) and SIGTERM (graceful termination by e.g. parent process), which are the 2 main ones you want to care about.

You can't handle SIGKILL as the process is killed by the OS immediately without getting a signal at all. And nobody uses SIGQUIT from what I know.